modules/Users/language/en_us.lang.php

```diff
@@ -685,9 +685,20 @@
     // for Google Token/Synchronization settings
     'LBL_GOOGLE_API_SETTINGS' => 'Google Account Synchronization',
     'LBL_GOOGLE_API_TOKEN' => 'Google API Token',
+    'LBL_GOOGLE_API_TOKEN_UNCONFIGURED' => 'UNCONFIGURED',
```

Comment:
Why capitalization?

Reply:
Hi @horus68, It was actually taken from the hard-coded value, which was already "UNCONFIGURED"; however, I could edit this to be "Not Configured"?

Reply:
For translators its better to not using ALL CAPS, as it allows translation memory to be used in full (previous translations can be reused automatically if they are 100% the same some other string already in use)